srml/contract/src/wasm/mod.rs

```diff
@@ -276,6 +276,9 @@ mod tests {
 		fn random_seed(&self) -> &H256{
 			&self.random_seed
 		}
+
+		fn deposit_event(&mut self, _data: Vec<u8>) {
```

Comment:
Maybe make it `unimplemented!()` to easily catch potential issues in the test code.

Reply:
yes I finally implement a test that checks charges